bin/unschedule-account-syncs

```diff
@@ -38,7 +38,7 @@ def main(dry_run, number, hostname, process):
         if process is not None:
             hostname = ':'.join([hostname, process])
         to_unschedule = db_session.query(Account.id). \
-                filter(Account.sync_host.like('{}%'.format(hostname)))
+            filter(Account.sync_host.like('{}%'.format(hostname)))
```

Comment:
Note that this file is still not formatted with black because it does not end up in `.py`. I will be addressing this on another PR.